ci/setup.sh

```diff
@@ -9,5 +9,9 @@ git submodule update --init --recursive
 # Install library compile deps.
 if ! [[ -z "$RUNNER_TEMP" ]]; then
   sudo apt-get update
-  sudo apt-get install lld
+  sudo apt-get install lld cmake
 fi
+
+# Build thirdparty deps
+cmake -Bthirdparty/flatbuffers/build thirdparty/flatbuffers
+cmake --build thirdparty/flatbuffers/build --target flatbuffers
```

Comment:
would be nice to have this in a build.rs that picked up changes to that dir but probably fine for now